### PR TITLE
Added parameters to all button listeners functions.

### PR DESCRIPTION
**Pull request by zkzhao333**

### CODE REVIEW

File: GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+     | Product ID   |  Price|
+     | :---:        | :---: |
+     | silver_subscription   | $1.99 |
+     | golden_subscription    | $4.99 |
+
+    Fill the other fields. Set them to "Active".
+
+13. Publish your APK to the Alpha channel. Wait 2-3 hours for Google Play to process the APK. If you don't wait for Google Play to process the APK, you might see errors where Google Play says that "this version of the application is not enabled for in-app billing" or something similar.
+
+14. Added Tester and License testing to your game on Play Console.
```
Comment (xinwei-gg) [inline GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/README.md]: nit: add a link to licensed tester since it's a play billing concept?

**Review by xinwei-gg — APPROVED** [review]

File: GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/README.md (hunk 1 above)
Comment (Qing451800) [reply GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/README.md]: This is an optional step that will make testing easier. It's still testable without doing this.

File: GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+## CHANGELOG
+2020-8-7: Inital release
+
+## Acknowledgements
+Most of the game assets are collected from [Kenney](https://www.kenney.nl/). Thanks Kenney for the amazing assets.
```
Comment (Qing451800) [inline GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/README.md]: nit: collected -> obtained

File: GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/README.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+6. Copy the application's public key (a base-64 string). You can find this in the "Services & APIs" section under "Licensing & In-App Billing".
+
+    GO BACK TO UNITY EDITOR
+7. Added Google Public Key to Obfuscated encryption keys. Check https://docs.unity3d.com/Manual/UnityIAPValidatingReceipts.html for instructions
+(note: If you can't see Unity IAP under Window in menu bar, you may need to reinstall UDP at **Assets/UDPInstaller/Editor/UDP.unitypackage**).
+8. Sign the App by creating a new key store at **File > Build Settings > Player Settings > Publishing Settings**. You can follow [this instruction](https://answers.unity.com/questions/326812/signing-android-application.html)
+9. Build the game.
+
+    GO BACK TO GOOGLE PLAY DEVELOPER CONSOLE
+10. Upload your APK to Googole Play for Beta Testing.
```
Comment (Qing451800) [inline GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/README.md:41]: Beta -> Alpha, or change Alpha to Beta below.

**Review by Qing451800 — APPROVED** [review]

**Pull request merged**